README.adoc

```diff
@@ -255,7 +255,7 @@ system/server.xml
 include::finish/system/src/main/liberty/config/server.xml[]
 ----
 
-Add the [hotspot=grpc file=1]`grpc` feature to the Liberty `server.xml` configuration file. This feature enables applications running on Liberty to provide gRPC services. Configure the [hotspot=grpcConfig file=1]`grpc` feature with the `target` attribute. If you want to learn more about configuration for the `grpc` feature, see the https://openliberty.io/docs/latest/reference/config/grpc.html[GRPC Server Properties^].
+Add the [hotspot=grpc file=1]`grpc` feature to the Liberty `server.xml` configuration file. This feature enables applications running on Liberty to provide gRPC services. Configure the [hotspot=grpcConfig file=1]`grpc` element with the `target` attribute to specify the remote gRPC service's endpoint that clients should connect to for communication. For simplicity in this guide, the `target` attribute is set to a wildcard (`*`), accepting any endpoint. If you want to learn more about configuration for the `grpc` element, see the https://openliberty.io/docs/latest/reference/config/grpc.html[GRPC Server Properties^].
```

Comment:
I do not think it is helpful to say that "*" is 'accepting any endpoint.
Is is not the case that a "*" can be used to add additional config (such as adding an interceptor)
to ALL the targets.